website_addthis/views/website_config_settings.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form_website" model="ir.ui.view">
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="website.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <div id="website_settings" position="inside">
+                <div class="col-xs-12 col-md-6 o_setting_box" id="addthis">
```

Comment:
Change col-xs-12 to col-12
see here: https://getbootstrap.com/docs/4.0/migration/